addons/binding/org.openhab.binding.nikohomecontrol/ESH-INF/thing/thing-types.xml

```diff
@@ -87,6 +87,12 @@
 				<description>Niko Home Control IP Interface Object ID</description>
 				<advanced>false</advanced>
 			</parameter>
+			<parameter name="invert" type="boolean" required="false">
```

Comment:
@kaikreuzer wdyt about such a feature?

Reply:
I guess you ask because I already clearly commented on that elsewhere before: For ESH/openHAB, it is clearly defined that 0% is UP and 100% is DOWN - this is the only accepted way that a binding should interpret the values. Therefore, an "invert parameter" only makes sense, if the remote system (here Niko Home Control) is so badly designed that it actually allows connecting motors in a fashion that one operates differently to others and you cannot determine this behavior from within the binding code. Is this the case here?

Reply:
@kaikreuzer indeed I had some vague memory about it

Reply:
@martinvw @kaikreuzer I had multiple testers trying out rollershutters (I don't have any myself). They came up with different values for up/down in their setup. I don't know there exact wiring, so can't judge if there is anything wrong with their wiring, but adding this parameter was suggested by them and solved the issue. I didn't find a way to get that info from the protocol, so made it a configuration parameter.
The solution was suggested to me in this post: https://community.openhab.org/t/niko-home-control/3482/150.

Reply:
Ok, thanks for the explanation - in this case I am fine with that feature (but you should maybe document that it is mean for exactly this and only this.

Reply:
@kaikreuzer @martinvw I haved added further clarification for the invert parameter.
However, to make absolutely sure I need the invert parameter, I have asked some more testing in the forum: https://community.openhab.org/t/niko-home-control-rollershutters-testing-required/43400?u=mherwege. If behaviour is the same for all test feedbacks I receive, I will remove the parameter.
I will mark this PR as WIP until then.

Reply:
@martinvw I just had someone testing today. It looks like I will not need the invert parameter after all. I am waiting for confirmation and will adapt the PR in that case. That should not take long.

Reply:
@martinvw I now removed the invert flag completely.  So this is good to be reviewed.